pkg/agent/cnf/podwatch/podwatch_controller_init.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2021 Antrea Authors
```

Comment:
I feel no need to add a separate file, and the func can be in podwatch_controller.go. But no strong opinion.